test/model-method.js

```diff
@@ -101,3 +99,106 @@ describe('ModelMethod', function() {
       });
   });
 });
+
+describe('ModelMethod - Loopback 3.0', function() {
+  var userModel;
+
+  beforeEach(givenLB3Workspace);
+  beforeEach(function(done) {
+    ModelDefinition.create(
+      {
+        name: 'user',
+        facetName: 'server'
+      },
+      function(err, result) {
+        if (err) return done(err);
+        userModel = result;
+        done();
+      });
+  });
+ 
+  it('add static method without isStatic flag to model definition', function(done) {
+    ModelMethod.create(
+      {
+        modelId: userModel.id,
+        name: 'testMethod',
+        isStatic: true
+      },
+      function(err) {
+        if (err) return done(err);
+        userModel.methods(function(err, list) {
+          if (err) return done(err);
+          expect(list).to.have.length(1);
+          expect(list[0]).to.have.property('name', 'testMethod');
+          expect(list[0]).to.not.have.property('isStatic');
+
+          var cfg = new ConfigFile({ path: 'server/models/user.json' });
+          cfg.load(function(err) {
+            if (err) return done(err);
+            var methods = cfg.data.methods;
+            expect(methods).to.be.an('object');
+            expect(methods).to.have.property('testMethod');
```

Comment:
Please add a check that the other name variant (`prototype.testMethod`) is not set.

```js
expect(methods).to.not.have.property('prototype.testMethod');
```

Just to be sure...

Please do the same in the other tests listed below too.